### DISCUSSION TITLE
Visual bug with dangerous pressure on a custom submarine

### DISCUSSION THREAD

**Discussion by NoPressF** [FakeFishGames/Barotrauma#15455] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

There's nothing wrong with the game, just a visual bug.

### Reproduction steps

1. My friend sent me his edited Remora submarine as mod
2. My friend added something else to the submarine, sent me the mod file again. When we arrived at the outpost and we buy a new updated submarine Remora, then at the beginning of the next round in the captain's navigation terminal it says "Dangerous pressure" and under some circumstances when we chose which next point to swim to, it showed us that our hull cannot be at depth and this was displayed for all points where you can swim, although we were in safe depth.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Are you sure it's just a visual bug? Pressure now kicks in only after a minute
![barodev small](https://github.com/user-attachments/assets/8488834f-536e-4ac3-8023-bdf2356ed3c3)

**Reply by Regalis11** [reply]
[Citation needed]

**Comment by NoPressF** [comment]
My friend and I swam 1000 meters down and nothing happened, this has happened several times already

**Comment by SomeRandomNoobKekeke** [comment]
That's probably because there's subs crush depth, and all the walls have individual crush depths, nav terminal uses subs crush deph

Ye, seems bugged, you just wanted to update sub midgame by rebuying it, right? I think i've done this before without such problems so something must've been broken

Also send save file and submarine file

**Reply by NoPressF** [reply]
Yes, I bought submarine in midgame

https://github.com/user-attachments/files/18276911/Bug.Report.zip

**Reply by Regalis11** [reply]
Thank you for the report and the save!

I gave this a test, but unfortunately could not find a way to reproduce the issue. The sub doesn't seem to be displaying the depth warning even in this level at the end of the hydrothermal wastes, nor any of the connections on the campaign map.
![image](https://github.com/user-attachments/assets/7cb0d3f4-d935-4ecb-bfb4-16b87df64773)
![image](https://github.com/user-attachments/assets/aee11889-07b4-4224-89f6-c6c91095de2d)
